docs/source/troubleshooting/lvrt-crash-dumps.rst

```diff
@@ -5,195 +5,169 @@ LabVIEW Core Dumps on Linux Real-Time
-#. Generate an NI MAX Technical Support Report and include this
-   when providing NI Support with the core dump.
+	.. code:: bash
+
+		admin@NI-cRIO-9036-01D5AED5:/var/local/natinst/log# tar -czf myCoreDump.tar.gz core_dump.\!usr\!local\!natinst\!labview\!lvrt
+		admin@NI-cRIO-9036-01D5AED5:/var/local/natinst/log# ls
+		SystemWebServer.log
+		core_dump.!usr!local!natinst!labview!lvrt
+		errlog.txt
+		lvlog01-20-20-21-27-51.txt
+		lvlog01-20-20-21-29-47.txt
+		lvlog01-20-20-21-35-13.txt
+		lvlog01-20-20-21-37-03.txt
+		lvlog01-20-20-21-42-12.txt
+		lvlog01-20-20-21-42-33.txt
+		lvrt_19.0_lvuser_cur.txt
+		lvrt_19.0_lvuser_log.txt
+		myCoreDump.tar.gz
+		admin@NI-cRIO-9036-01D5AED5:/var/local/natinst/log#
+
+#. Copy the core dump archive to a host system via a supported file transfer method (FTP, sFTP, WebDAV, etc).
+
+#. Generate an NI MAX Technical Support Report and include this when providing NI Support with the core dump.
+
 
 Confirming Core Dumps are Generated
 -----------------------------------
 
-| To confirm core dumps are properly configured, it's possible to
-  force a crash of the LabVIEW Real-Time process for testing purposes.
+To confirm core dumps are properly configured, it's possible to force a crash of the LabVIEW Real-Time process for testing purposes.
 
 #. Configure a target for Core Dumps as described above.
-#. Open a terminal on the target (SSH or Serial) and log in as the admin
-   user. 
+
+#. Open a terminal on the target (SSH or Serial) and log in as the ``admin`` user.
+
 #. Run the following command:
 
-   .. code:: bash
+	.. code:: bash
+
+		killall -3 lvrt
 
-      killall -3 lvrt
+	This command will send a SIGSEGV to the lvrt process.
 
-   This command will send a SIGSEGV to the lvrt process.
 #. Confirm that the lvrt process crashed by running the following
-   command:
+	command:
+
+	.. code:: bash
+
+		ps -aux | grep lvrt
+
+	If lvrt crashed, there should be something similar to the following result, noting the CRASHED_AND_RESTART message:
+
+	.. code:: bash
+
+		1707 admin      0:00 {lvrt-daemon} /bin/sh /etc/init.d/lvrt-daemon
+		2408 admin      0:00 /bin/su -- lvuser -l -c /etc/init.d/lvrt-wrapper CRASHED_AND_RESTART /var/run/lvrt_wrapper.pid false
+		2409 lvuser     0:00 {MainAppThread} ./lvrt
+		2473 admin      0:00 grep lvrt
+
+#. Confirm that a core dump is now present in ``/var/local/natinst/log/``. The core dump should have a name similar to ``core_dump.!usr!local!natinst!labview!lvrt``.
+
+	.. code:: bash
+
+		admin@NI-cRIO-9036-01D5AED5:~# ls /var/local/natinst/log
+		SystemWebServer.log
+		core_dump.!usr!local!natinst!labview!lvrt
+		errlog.txt
+		lvlog01-20-20-21-27-51.txt
+		lvlog01-20-20-21-29-47.txt
+		lvlog01-20-20-21-35-13.txt
+		lvlog01-20-20-21-37-03.txt
+		lvrt_19.0_lvuser_cur.txt
+		lvrt_19.0_lvuser_log.txt
+
+#. Remove the core dump before proceeding to reproduce the actual crash.
+
+
+Disabling LVRT Core Dumps
+=========================
 
-   .. code:: bash
+.. versionadded:: 9.1
```

Comment:
Since this was technically added in lvrt and not nilrt, do we want to use the lvrt version here? I'm fine either way.

Reply:
Yeah; it probably makes sense to note the version of the `ni-labview-realtime` IPK. Especially since I think this behavior will differ depending on what version of the LV feeds the target is configured to use.

Will fix.